pydrive2/fs/spec.py

```diff
@@ -449,11 +449,12 @@ def ls(self, path, detail=False):
                     }
                 )
             else:
+                s = item.get("fileSize")
```

Comment:
s is a bad name, our linters in other projects would've complained
```suggestion
                size = item.get("fileSize")
```

Reply:
yep, resolved (need at some point migrate to py-template)